### PR TITLE
Landing page updates

### PR DESCRIPTION
**Pull request by Michael-Turkington**

### CODE REVIEW

File: app/assets/stylesheets/components/_index.scss [hunk 1]
```diff
@@ -9,6 +9,10 @@
 @import "main_button";
 @import "embeded_video";
 @import "youtube_video_banner";
+<<<<<<< HEAD
+@import "kitchen_card";
```
Comment (Haumer) [inline app/assets/stylesheets/components/_index.scss:13]: my guess is you want to keep this one, so remove lines 12, 14 and 18

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -1,6 +1,10 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
 
+<<<<<<< HEAD
+    <%= link_to root_path, class: "navbar-brand" do %>
```
Comment (Haumer) [inline app/views/shared/_navbar.html.erb:4]: i guess only the indentation changed here? 

should be able to just remove lines 3, 5, 7

File: db/schema.rb [hunk 3]
```diff
@@ -10,7 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
+ActiveRecord::Schema.define(version: 2020_09_12_095807) do
```
Comment (Haumer) [inline db/schema.rb:14]: this is probs the most tricky one, seems like you have an earlier migration than the one on the master... 
if that is what you want you can keep this one and remove lines 13 and 15 -17 (dont keep both ActiveRecord::Schemas)

File: app/assets/stylesheets/components/_index.scss (hunk 1 above)
Comment (Michael-Turkington) [reply app/assets/stylesheets/components/_index.scss:13]: I don't think so, it's not just this component_index. There was lots of pages missing from the last push/pull request

**Pull request merged**